src/RichTextEditor.js

```diff
@@ -154,14 +155,20 @@ export default class RichTextEditor extends Component {
         case messages.CONTENT_FOCUSED:
           this.contentFocusHandler && this.contentFocusHandler();
           break;
-        case messages.SELECTION_CHANGE:
+        case messages.SELECTION_CHANGE: {
           const items = message.data.items;
-          this.state.listeners.map((listener) => listener(items));
-          break
-        case messages.CONTENT_CHANGE:
```

Comment:
It's strange how I missed that. I'll turn the `messages.CONTENT_CHANGE` case back.